Client/Frontend/Home/HomepageViewModel.swift

```diff
@@ -50,7 +50,18 @@ class HomepageViewModel: FeatureFlaggable {
 
     let nimbus: FxNimbus
     let profile: Profile
-    var isZeroSearch: Bool
+    var isZeroSearch: Bool {
```

Comment:
@lmarceau, what's a zero search? (like nothing in search)

Reply:
It's a term that is used for telemetry, if we go up the chain where this param is passed in it's explained a bit. So that is in `showHomepage` method.

```
    /// Show the home page
    /// - Parameter inline: Inline is true when the homepage is created from the tab tray, a long press
    /// on the tab bar to open a new tab or by pressing the home page button on the tab bar. Inline is false when
    /// it's the zero search page, aka when the home page is shown by clicking the url bar from a loaded web page.
```

Reply:
@lmarceau how is inline homepage related to zero search, sorry I am not quite following it

Reply:
Its for tracking 

```
// Firefox home page tracking i.e. being shown from awesomebar vs bottom right hamburger menu
        let trackingValue: TelemetryWrapper.EventValue = isZeroSearch
```

Reply:
Found it!!